coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -9,6 +13,10 @@ class DocumentationComment:
     The DocumentationComment holds information about a documentation comment
     inside source-code, like position etc.
     """
+    _ParseMode = enum("DESCRIPTION", "PARAM", "RETVAL")
+    Param = namedtuple('Param', 'name, desc')
+    RetVal = namedtuple('RetVal', 'desc')
+    Desc = namedtuple('Desc', 'desc')
```

Comment:
though this is not officially supported, but I think sphinx allows/interpretes docstrings above variables as documentation:
```
"""
Desc
"""
Desc = namedtuple(...)
```